src/sentry/lang/native/unreal.py

```diff
@@ -112,3 +112,11 @@ def merge_unreal_context_event(unreal_context, event, project):
         'name': 'sentry.unreal.crashreporter',
         'version': runtime_prop.pop('crash_reporter_client_version', '0.0.0')
     }
+
+
+def merge_unreal_logs_event(unreal_logs, event):
+    setdefault_path(event, 'breadcrumbs', 'values', value=[])
+    breadcrumbs = event['breadcrumbs']['values']
+
+    for log in unreal_logs:
+        breadcrumbs.append(log)
```

Comment:
This will append the the logs as they are provided by `symbolic-unreal`, but they don't fully match the [Breadcrumb interface](https://docs.sentry.io/development/sdk-dev/interfaces/breadcrumbs/). Specifically, `Unreal4LogEntry` has a `component` field which will be stripped from breadcrumbs. Should that actually be `category`?